docs/Gemfile.lock

```diff
@@ -1,16 +1,12 @@
-PATH
-  remote: ..
-  specs:
-    bootstrap-bookingsync-sass (1.0.5)
-      bootstrap-sass (>= 3.3.5)
-
 GEM
   remote: https://rubygems.org/
   specs:
     adsf (1.2.0)
       rack (>= 1.0.0)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.7.6)
       execjs
+    bootstrap-bookingsync-sass (2.0.0)
+      bootstrap-sass (>= 3.3.5)
```

Comment:
The adjustment you've made seems to be more of a preparation to use bootstrap 4.
As noted on a `bootstrap-sass` gem `This is Bootstrap 3. For Bootstrap 4 use the Bootstrap rubygem if you use Ruby, and the main repo otherwise.`
Since we already bump version, perhaps it makes sense to update dependencies to rely on bootstrap 4 here?

Reply:
Using Bootstrap 4 is a massive change and not for this time